android/src/main/java/com/swmansion/gesturehandler/core/GestureHandler.kt

```diff
@@ -108,7 +114,18 @@ open class GestureHandler<ConcreteGestureHandlerT : GestureHandler<ConcreteGestu
         return true
       }
     }
-    return false
+
+    if (this !is TapGestureHandler) {
+      return false
+    }
```

Comment:
This looks weird to me. Why not just make this method `open` and override it in `TapGestureHandler`?

Reply:
Done in [ed702f8](https://github.com/software-mansion/react-native-gesture-handler/pull/2759/commits/ed702f8b203bd641dee038628a5ac0ee1e01dab5)